ibm_catalog.json

```diff
@@ -114,37 +305,36 @@
+                            "version": ">=1.7.0"
                         }
                     ],
                     "architecture": {
                         "features": [
                             {
-                                "description": "PowerVS service with all required components",
-                                "title": "Power Virtual Server workspace"
+                                "description": "PowerVS Workspace with all required components",
+                                "title": "Power Virtual Server Workspace"
                             },
                             {
-                                "description": "Proxy service to reach public internet from PowerVS workspace",
+                                "description": "Proxy service to reach public internet from PowerVS Workspace",
                                 "title": "Internet proxy on VPC"
                             },
                             {
-                                "description": "NFS server, NTP forwarder, and DNS forwarder reachable from PowerVS workspace",
+                                "description": "NFS server, NTP forwarder, and DNS forwarder reachable from PowerVS Workspace",
                                 "title": "Additional management configurations on VPC"
                             }
                         ],
                         "diagrams": [
                             {
                                 "diagram": {
                                     "caption": "Power infrastructure for deployable architectures",
-                                    "url": "https://raw.githubusercontent.com/terraform-ibm-modules/terraform-ibm-powervs-infrastructure/main/reference-architectures/standard/deploy-arch-ibm-pvs-inf-standard.svg",
+                                    "url": "https://raw.githubusercontent.com/terraform-ibm-modules/terraform-ibm-powervs-infrastructure/main/reference-architectures/extension/deploy-arch-ibm-pvs-inf-extension.svg",
                                     "type": "image/svg+xml"
                                 },
-                                "description": "Building upon the VPC service that were previously created when you deployed Secure infrastructure on VPC for regulated industries, Power Infrastructure for deployable architectures creates a Power Virtual Server workspace and connects it with VPC services. Proxy service for public internet access from PowerVS workspace is configured. \n\nAdditional management components (NFS server, NTP forwarder, and DNS forwarder) may be installed on VPC optionally."
+                                "description": "Deploys  Power Virtual Server workspace and connects it with VPC services. Proxy service for public internet access from PowerVS workspace is configured. \n\nAdditional management components (NFS server, NTP forwarder, and DNS forwarder) may be installed on VPC optionally."
```

Comment:
"description": "Building upon the VPC service that were previously created when you deployed 'Secure infrastructure on VPC for regulated industries' or 'Power Infrastructure for deployable architecture' in a full-stack variation 'PowerVS Workspace', extension variation of ''PowerVS Workspace' creates a Power Virtual Server workspace and connects it with VPC services. \n\nProxy service for public internet access from PowerVS workspace is configured. \n\nYou can optionally reconfigure management components on VPC (such as NFS server, NTP forwarder, and DNS forwarder)."

Reply:
Done